sdk/src/main/java/com/supenta/flitchio/sdk/FlitchioController.java

```diff
@@ -292,39 +298,37 @@ public boolean onCreate() throws FlitchioManagerDependencyException {
     }
 
     /**
-     * Register a {@link FlitchioListener} to receive callbacks. If you use this
+     * Register a {@link FlitchioEventListener} to receive callbacks. If you use this
      * {@link FlitchioController} in an {@link Activity}, this should be called in your Activity's
      * onResume() (hence the name). If you use this {@link FlitchioController} in a {@link Service},
      * this can be called right after {@link #onCreate()}. You only need to call this if you declare
-     * a {@link FlitchioListener}.
+     * a {@link FlitchioEventListener}.
      *
-     * @param listener The listener.
-     * @param handler  The handler associated to the thread on which the callbacks will happen.
+     * @param eventListener The eventListener.
+     * @param handler       The handler associated to the thread on which the callbacks will happen.
      * @since 0.5.0
      */
-    public void onResume(FlitchioListener listener, Handler handler) {
+    public void onResume(FlitchioEventListener eventListener, FlitchioStatusListener statusListener, Handler handler) {
```

Comment:
It is a detail, but considering that StatusListener is likely to be more frequently used than EventListener (I think), I'd put it as first parameter.